pallets/dkg-proposals/src/tests.rs

```diff
@@ -517,3 +521,78 @@ fn proposal_expires() {
 		})]);
 	})
 }
+
+#[test]
+fn should_get_initial_proposers_from_dkg() {
```

Comment:
This test checks if the proposers are set after the DKG authorities are initialized in the genesis session

Reply:
Can we add tests to validate that they change when sessions update?